app/commands/jobs.rb

```diff
@@ -20,7 +21,7 @@ def more(id)
     end
 
     def call(message)
-      dataset.each_with_index do |job, i|
+      repo.all.each do |job|
```

Comment:
Ive been thinking about using some kind of formatter to represent data, wdyt?

Reply:
yep, i think it's necessary to separate data formatting and data fetching

p.s. sorry for long answer delay